OJP/OJP_Trips.xsd

```diff
@@ -122,6 +135,21 @@
 					<xs:documentation>For each mode in this list a separate monomodal trip shall be found - in addition to inter-modal solutions.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
+			<xs:element name="PassengerCategory" type="PassengerCategoryEnumeration" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Sequence of all passenger categories, for which this search shall be conducted. In rare cases additional offerings may be available (e.g., demand responsive service with certain areas being reserved for seniors).</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="AccessModeList" type="AccessModesListOfEnumerations" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Access modes that are available to the traveller for the trip. The requester can tell the system what he might use as an access mode (e.g., car, motorcycle, ski) and it's the system's responsibility to know which access modes are supported or required by each service. E.g., if one goes by car, then only ferry and carTransportRail may be possible for public transport. If the traveller has skis or a snowboard with him, then dragLifts can show up in the results to a search. When nothing is provided in the time table, then foot and bicycle should be assumed by the trip planner.</xs:documentation>
```

Comment:
Shouldn't the filtering behavior based on *AccessModeList* be more clearly stated here? If Goppenstein-Kandersteg did support car but not bicycle, what would happen if I had both of them in the *AccessfModeList*? Would the journey be excluded (not possible to transport all of them) or included? What should be the behavior if I want to go from Goppenstein to the top of a drag lift above Kandersteg and I have car and ski in the *AccessModeList*? Should the algorithm be so intelligent to propose Goppenstein-Kandersteg by train, followed by a car drive to the drag lift, followed by the drag lift (transporting the skis but not the car)? Or has every leg to be queried separately? (But this shouldn't be done according to the implementors.)